libcxx/test/libcxx/atomics/atomics.syn/wait.pass.cpp

```diff
@@ -0,0 +1,32 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// UNSUPPORTED: c++03, c++11, c++14, c++17
+// UNSUPPORTED: no-threads
+// XFAIL: availability-synchronization_library-missing
+// This bug was first fixed in LLVM 19
+// XFAIL: using-built-library-before-llvm-19
```

Comment:
Maybe just turn this into `UNSUPPORTED` since the `apple-backdeployment-hardening` looks like it would require  very tedious checking of the software versions and maybe even hardware.